draft-ietf-rats-eat.md

```diff
@@ -573,17 +573,23 @@ A Device Identifier URN is registered for UEIDs. See {{registerueidurn}}.
 
 ### Semi-permanent UEIDs (SUEIDs)
 
-An SEUID is of the same format as a UEID, but it MAY change to a different value on device life-cycle events.
-Examples of these events are change of ownership, factory reset and on-boarding into an IoT device management system.
+An SUEID has the same format, characteristics and requirements as a UEID, but MAY change to a different value on entity life-cycle events.
 An entity MAY have both a UEID and SUEIDs, neither, one or the other.
 
+Examples of life-cycle events are change of ownership, factory reset and on-boarding into an IoT device management system.
+It is beyond the scope of this document to specify particular types of SUEIDs and the life-cycle events that trigger their change.
+An EAT profile MAY provide this specification.
```

Comment:
Since EUI and IMEI labels come from a registry, they can't change during a life-cycle event.
It seems that really only the RAND type makes any sense for the SUEID.